src/components/ContactProfile/ProfileContent.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/macro'
+
+const contacts = {
+  firstName: 'Max',
+  lastName: 'Mustermann',
+  company: 'Universal GmbH',
+  phone: '0176-123123123',
+  mail: 'max.mustermann@universal.de',
+  street: 'ABC-Straße 105',
+  zip: '20239',
+  city: 'Hamburg',
+  note: 'Wv Montag',
+  category: 'pot. Kunde',
+}
+
+export default function ProfileContent() {
```

Comment:
ProfileContent doesn't have any props. However in ContactProfile you use it like this ` return <ProfileContent contacts={contacts} />` with a contacts-prop. That looks incorrect to me.

Reply:
Fixed it! Thanks for the hint! :)